challenges/MeetPandya Coding Challenge

```diff
@@ -0,0 +1,46 @@
+#Back-end Coding challenge for Headstorm
+
+#Meet Pandya
+#email id : mnpandya10@gmail.com
+
+import flask
+import random
+from flask import jsonify, request
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+def rand (start, end, num):
+    numbers = []
+    if num!=500:
+        print("Numbers are not 500")
+    for i in range(num):
+        numbers.append(random.randint(start,end))
+    return numbers
+
+numbers_incount = rand(1,1000,500)
+
+@app.route('/', methods=['GET'])
+def message():
+    return jsonify({'message' : 'Coding Challenge for Headstorm'})
+
+@app.route('/data', methods=['GET'])
+def returnAllNum():
+    numbers_incount.sort()
```

Comment:
How would you change your code if we discover that this application is write-heavy and read-light, or in other words, if POST is called frequently but GET is only called rarely. Assume we need to optimize this system so writes can be as fast as possible but still fulfill the requirements given for sorting.

Reply:
After getting info into handler, we can dump it to JSON and return it back